Resources/Prototypes/Entities/items/clothing/IDs.yml

```diff
@@ -3,46 +3,74 @@
   id: IDCardStandard
   name: Identification Card
   description: A card necessary to access various areas aboard the station
+  abstract: true
   components:
   - type: Sprite
-    sprite: Clothing/id_cards.rsi
-    state: civilian
-  - type: Icon
-    sprite: Clothing/id_cards.rsi
-    state: civilian
+    drawdepth: FloorObjects
   - type: Clothing
     Slots:
     - idcard
     sprite: Clothing/id_cards.rsi
-    HeldPrefix: civilian
+    HeldPrefix: default
   - type: IdCard
   - type: Access
 
+#IDs with layers
+
 - type: entity
   parent: IDCardStandard
-  id: CommandIDCard
-  name: Command Identification Card
+  id: AssistantIDCard
+  name: Assistant ID Card
   components:
-  - type: Clothing
-    HeldPrefix: gold
   - type: Sprite
-    state: gold
+    sprite: Clothing/id_cards.rsi
```

Comment:
You can just specify this in `IDCardStandard` then you don't have to put `sprite: Clothing/id_cards.rsi` in every ID card.